.github/workflows/001-cargo-install-ubuntu-make-build.yml

```diff
@@ -98,4 +98,4 @@ jobs:
         # The goal is for this to be easy to find from the GitHub dashboard.
         # Instead of adding more commands here, consider another make target or a new YAML file
         # named with a good name.
-        run: make build
+        run: make musl build
```

Comment:
if build needs musl, build should depend on musl :)

Reply:
You are right, but I hate that we are calling it every time locally (even thought it's cheap). I'll update